### PR TITLE
[TEST] Add tensor query tests @open sesame 08/27 19:27

### PR DESCRIPTION
**Pull request by gichan-jang**
Add tensor query tests.
Line coverage increased from 6.6% to 79.1%.

Signed-off-by: gichan <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: tests/nnstreamer_query/unittest_query.cc [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+  EXPECT_EQ (TRUE, bool_val);
+
+  /* Set properties of query client */
+  g_object_set (client_handle, "src-host", "127.0.0.2", NULL);
+  g_object_get (client_handle, "src-host", &str_val, NULL);
+  EXPECT_STREQ ("127.0.0.2", str_val);
+  g_free (str_val);
+
+  g_object_set (client_handle, "src-port", 5001U, NULL);
+  g_object_get (client_handle, "src-port", &int_val, NULL);
```
Comment (anyj0527) [inline tests/nnstreamer_query/unittest_query.cc:189]: Should use `uint_val`?

File: tests/nnstreamer_query/unittest_query.cc [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+  EXPECT_EQ (5001U, int_val);
```
Comment (jaeyun-jung) [inline tests/nnstreamer_query/unittest_query.cc:190]: Is it necessary? If so, you should fix build error.
Comparison signed and unsigned value makes build error on app-test (e.g., line 190, 199)

File: tests/nnstreamer_query/unittest_query.cc (hunk 1 above)
Comment (gichan-jang) [reply tests/nnstreamer_query/unittest_query.cc]: Thanks. Updated.

File: tests/nnstreamer_query/unittest_query.cc (hunk 2 above)
Comment (gichan-jang) [reply tests/nnstreamer_query/unittest_query.cc]: No. My bad. Updated.

**Review by jaeyun-jung — APPROVED** [review]

**Review by myungjoo — APPROVED** [review]
LGTM

**Pull request merged**